### PR TITLE
Add vue3 upgrade and help video news items

### PR DESCRIPTION
**Pull request by r-ash**
Reviving this NEWS now we are doing some more user facing stuff. 

This adds a post about vue3 upgrade and the link to help videos :sunglasses:

### CODE REVIEW

File: content/posts/vue3.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Vue3 upgrade"
+date: 2023-09-24
+version: ["hint v3.0.0"]
```
Comment (EmmaLRussell) [inline content/posts/vue3.md:4]: This is the same version as the Help videos, but nominal publication date is different.

**Review by EmmaLRussell — COMMENTED** [review]
I'm sure this is all fine, I'm just getting weird formatting when I run it locally (with hugo v0.120.1). I have a feeling I might have had this issue the last time I looked at it though!
Screenshot from 2023-10-31 18-14-01

File: content/posts/vue3.md (hunk 1 above)
Comment (r-ash) [reply content/posts/vue3.md:4]: Oh yeah I just guessed at when we deployed vue3.. that was a bit ago? Didn't think it was actually very important to get that spot on. Help is v3.3.0

**Comment by r-ash** [comment]
> I'm sure this is all fine, I'm just getting weird formatting when I run it locally (with hugo v0.120.1). I have a feeling I might have had this issue the last time I looked at it though! Screenshot from 2023-10-31 18-14-01

Hmm that is weird, I was not seeing that locally. Do you have the submodules initialised and updated?

File: content/posts/vue3.md (hunk 1 above)
Comment (EmmaLRussell) [reply content/posts/vue3.md:4]: No, it probably doesn't matter too much

**Comment by EmmaLRussell** [comment]
> > I'm sure this is all fine, I'm just getting weird formatting when I run it locally (with hugo v0.120.1). I have a feeling I might have had this issue the last time I looked at it though! Screenshot from 2023-10-31 18-14-01
> 
> Hmm that is weird, I was not seeing that locally. Do you have the submodules initialised and updated?

Yeah, I updated submodules :shrug:
Like I say, i have a memory of it happening before, so I think it is just my problem

**Review by EmmaLRussell — APPROVED** [review]

File: content/posts/help-videos.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Help videos now available"
+date: 2023-10-31
+version: ["hint v3.3.0"]
+pull_request: https://github.com/mrc-ide/hint/pull/877
+tags: ["Feature"]
+---
+
+2 years ago we created some instruction videos about how to use Naomi. We've now added a link from the "Online support" menu in the header to the videos. We've plans to update and expand these video tutorials going forwards so stay tuned for updates!
```
Comment (M-Kusumgar) [inline content/posts/help-videos.md]: I think :laughing: also would be nice to just have a link to the youtube channel here too no?
```suggestion
2 years ago we created some instruction videos about how to use Naomi. We've now added a link from the "Online support" menu in the header to the videos. We've plans to update and expand these video tutorials going forward so stay tuned for updates!
```

**Review by M-Kusumgar — APPROVED** [review]
Looks good! see minor comment

File: content/posts/help-videos.md (hunk 2 above)
Comment (r-ash) [reply content/posts/help-videos.md]: Thanks, I've added that link now

**Pull request merged**